qiita_pet/handlers/websocket_handlers.py

```diff
@@ -70,3 +72,26 @@ def on_close(self):
         yield Task(self.toredis.unsubscribe, self.channel)
         self.r_client.delete('%s:messages' % self.channel)
         self.redis.disconnect()
+
+
+class SelectSamplesHandler(WebSocketHandler, BaseHandler):
+    @authenticated
+    def on_message(self, msg):
```

Comment:
Thanks a lot!

Yoshiki.

On (Apr-16-15|10:20), Joshua Shorenstein wrote:
> > @@ -70,3 +72,26 @@ def on_close(self):
> >          yield Task(self.toredis.unsubscribe, self.channel)
> >          self.r_client.delete('%s:messages' % self.channel)
> >          self.redis.disconnect()
> > +
> > +
> > +class SelectSamplesHandler(WebSocketHandler, BaseHandler):
> > +    @authenticated
> > +    def on_message(self, msg):
> 
> ok, just wanted clarification since we're in transition of code guidelines. Will add.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1056/files#r28530242